seedDB/README.md

```diff
@@ -1,25 +1,7 @@
-Files to seed a new cBioPortal database.
+These files are MySQL database dump for seeding a new instance of the cBioPortal DB. They contain all the necessary background data for a properly functioning cBioPortal website, including cancer types, gene, uniprot-mappings, drug, pdb and network data. Data is compatible with release **1.3.1** of cBioPortal.  
```

Comment:
I am not sure that the database schema number and the cBioPortal codebase version number are coupled to each other. Already the public release at cbioportal.org reports a 1.3.2 version number, while the cgds.sql script in rc seems to be at 1.3.0 and migration.sql is at 1.3.1 [incidentally I think that is a mistake --- I would argue that anytime migration.sql gets incremented there should be a matching increment to cgds.sql].  But the main point is that the seed db will track a version of the schema -- which will often be different than the codebase version.
I would maybe add an explanation of the two different versions:
- codebase [from pom.xml I think]
- DB_SCHEMA_VERSION [from cgds/migration, and checked against portal.properties db.version